src/scripts/main.js

```diff
@@ -1,3 +1,96 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  // максимальна кіслькість рядків і стовців
+  const maxRow = 10;
+  const maxColumn = 10;
+  const minRow = 2;
+  const MinColumn = 2;
+
+  const div = document.querySelector('.container');
+  const appendRow = div.querySelector('.append-row ');
+  const removeRow = div.querySelector('.remove-row');
+  const appendColumn = div.querySelector('.append-column');
+  const removeColumn = div.querySelector('.remove-column');
+  const table = div.querySelector('.field');
+
+  // функція видалення якщо кількість рядків зменшилась
+  function updateRowButtons() {
+    removeRow.disabled = table.rows.length <= minRow;
+    appendRow.disabled = table.rows.length >= maxRow;
+  }
+
+  function updateColumnButtons() {
+    removeColumn.disabled = table.rows[0].cells.length <= MinColumn;
```

Comment:
The variable `MinColumn` is used here, which is inconsistent with the camelCase naming convention. It should be `minColumn` to match the declaration and maintain consistency.